seacrowd/sea_datasets/culturax/culturax.py

```diff
@@ -0,0 +1,146 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+from urllib.parse import urljoin
+
+import datasets
+from pyarrow import parquet as pq
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks
+
+_CITATION = """\
+@article{nguyen2023culturax,
+    author    = {Thuat Nguyen and Chien Van Nguyen and Viet Dac Lai and Hieu Man and Nghia Trung Ngo and Franck Dernoncourt and Ryan A. Rossi and Thien Huu Nguyen},
+    title     = {CulturaX: A Cleaned, Enormous, and Multilingual Dataset for Large Language Models in 167 Languages},
+    journal   = {arXiv preprint arXiv:2309.09400},
+    year      = {2023},
+    url       = {https://arxiv.org/abs/2309.09400},
+}
+"""
+
+_DATASETNAME = "culturax"
+_DESCRIPTION = """\
+CulturaX is a comprehensive multilingual dataset comprising 6.3 trillion tokens across 167
+languages, designed for large language model development. It incorporates an advanced
+cleaning and deduplication process, including language identification and fuzzy
+deduplication with MinHash, to ensure high-quality data for model training. The dataset,
+which spans 16TB in parquet format and 27TB when unpacked, is a combination of the latest
+mC4 and OSCAR corpora, emphasizing non-English languages to support multilingual model
+training. For data cleaning validation, CulturaX employs a SentencePiece tokenizer and
+KenLM language models, utilizing recent Wikipedia dumps for perplexity scoring.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/uonlp/CulturaX"
+_LICENSE = """
+    The licence terms for CulturaX strictly follows those of mC4 and OSCAR. \
+    Please refer to both below licenses when using this dataset. \
+    - mC4 license: https://huggingface.co/datasets/allenai/c4#license \
+    - OSCAR license: https://huggingface.co/datasets/oscar-corpus/OSCAR-2301#licensing-information \
+"""
+_BASE_URL = "https://huggingface.co/datasets/uonlp/CulturaX/resolve/main/{lang}/"
+
+_SUPPORTED_TASKS = [Tasks.SELF_SUPERVISED_PRETRAINING]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class CulturaXDataset(datasets.GeneratorBasedBuilder):
+    """CulturaX subset for SEA languages."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "ssp"
+    SUBSETS = ["id", "jv", "km", "lo", "tl", "min", "my", "su", "th", "vi", "ms", "ceb", "war", "cbk", "bcl"]
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{subset}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME} {subset} source schema",
+            schema="source",
+            subset_id=subset,
+        ) for subset in SUBSETS
+    ] + [
```

Comment:
Can you rerun `make check_file=...` here? I reran this locally and this part of the source changed formatting.

Reply:
I actually did, and became a one-liner. But, the formatting in my opinion is better this way. What do you think?

Reply:
Oh, maybe we had different settings? Mine's just a small change:

```diff
diff --git a/seacrowd/sea_datasets/culturax/culturax.py b/seacrowd/sea_datasets/culturax/culturax.py
index c023a8a..920bf12 100644
--- a/seacrowd/sea_datasets/culturax/culturax.py
+++ b/seacrowd/sea_datasets/culturax/culturax.py
@@ -61,7 +61,8 @@ class CulturaXDataset(datasets.GeneratorBasedBuilder):
             description=f"{_DATASETNAME} {subset} source schema",
             schema="source",
             subset_id=subset,
-        ) for subset in SUBSETS
+        )
+        for subset in SUBSETS
     ] + [
         SEACrowdConfig(
             name=f"{_DATASETNAME}_{subset}_seacrowd_ssp",
```

```suggestion
        ) 
        for subset in SUBSETS
    ] + [
```

Reply:
I'm not sure why, but I changed it like yours now.